tutorial/20_recipes/008_specify_params.py

```diff
@@ -0,0 +1,121 @@
+"""
+.. _specify_params:
+
+Specify Hyperparameters
+=======================
+
+Sometimes, it is natural for you to try some experiments with your out-of-box hyperparameters.
+For example, you have some specific sets of hyperparameters to try in your mind before using Optuna for the best hyperparameters.
+
+First scenario
```

Comment:
How about adding some words to make the section title more descriptive? It may be a bit difficult for readers to see the difference at a glance. The following is just an example:

```suggestion
First scenario: Add hyperparameter sets to be evaluated
```